### PR TITLE
feat: add configuration option for ipv4 or ipv6, defaults to ipv4

### LINKED ISSUE #91: UdpWriter does not support IPv6
__Proposal:__
I would like to be able to use the UdpWriter not only with IPv4 but also with IPv6.

__Current behavior:__
Currently only IPv4 is supported due to the nature of [socket_create](https://www.php.net/manual/en/function.socket-create.php)

__Desired behavior:__
I would like to have a way to configure the client to use either IPv4 (maybe default) or IPv6.

__Alternatives considered:__
Alternative could be some kind of auto detection if IPv6 is available but i think this might be a bit too much.

__Use case:__
I want to use the UdpWriter in an infrastructure environment where IPv6 is used.

### PR DESCRIPTION
**Pull request by kevin0x90**
Closes #91 

## Proposed Changes

New configuration option for IPv4 or IPv6

## Checklist

- [x] CHANGELOG.md updated
- [x] Rebased/mergeable
- [x] A test has been added if appropriate
- [x] `make test` completes successfully
- [x] Commit messages are in [semantic format](https://seesparkbox.com/foundry/semantic_commit_messages)
- [x] Sign [CLA](https://influxdata.com/community/cla/) (if not already signed)

### CODE REVIEW

File: src/InfluxDB2/UdpWriter.php [hunk 1]
```diff
@@ -84,15 +84,35 @@ protected function writeSocket($payload)
...
      * Create (if not exists) socket to write UDP datagrams
      * @return false|resource
+     * @throws \Exception
      */
     protected function getSocket()
     {
         if (empty($this->socket)) {
-            $this->socket = socket_create(AF_INET, SOCK_DGRAM, SOL_UDP);
+
+            $this->socket = socket_create($this->getConfiguredInetVersion(), SOCK_DGRAM, SOL_UDP);
```
Comment (bednar) [inline src/InfluxDB2/UdpWriter.php:93]: Please remove blank line to satisfy code style check:

```suggestion
            $this->socket = socket_create($this->getConfiguredInetVersion(), SOCK_DGRAM, SOL_UDP);
```

**Review by bednar — CHANGES_REQUESTED** [review]
Thanks for your PR 👍 
There are a few requirements that must be be satisfy before we accept the PR:

1. Update `CHANGELOG.md`
1. Update `README.md` with information how to configure ipv6:
     image
1. Please add a documentation for new configuration option (`ipVersion`) in a Client constructor:
https://github.com/influxdata/influxdb-client-php/blob/6640e711358086c6056727b785cd90fb3f5d4cb6/src/InfluxDB2/Client.php#L58


and also:

File: src/InfluxDB2/UdpWriter.php (hunk 1 above)
Comment (kevin0x90) [reply src/InfluxDB2/UdpWriter.php:93]: Will do 👍

**Review by bednar — APPROVED** [review]
Thanks again for your PR 👍

**Pull request merged**